settings/blt.settings.php

```diff
@@ -64,9 +64,8 @@
 }
 
 $repo_root = dirname(DRUPAL_ROOT);
-
 if (!isset($site_path)) {
-  $site_path = \Drupal::service('site.path');
+  $site_path = 'default';
```

Comment:
This is probably unnecessary if settings.php files are included correctly by Drupal and should probably fail if not rather than assuming the `$site_path` is `default` without additional checks or context.

Reply:
Yeah, I'm happy if I see a throw error here, just to be sure this file was included in a way that site_path is set.

Reply:
I just reverted the most recent commit, so there's no more check or default value. I think it's perfectly safe to assume this is set by core, as even core itself does this. I don't see how throwing an additional error would add value.